# Define the goal
float32[] target_pos # desired position of the head  
---
# Define the result
bool pos_reached # True if desired position has is reached
---
# Define a feedback message
float32[] current_pos # current position of the head
